cernopendata/modules/fixtures/data/docs/atlas-open-data-for-research-release-2024/atlas-open-data-for-research-release-2024.md

```diff
@@ -0,0 +1,34 @@
+
+“Open access is a core value of CERN and the ATLAS Collaboration,” says Andreas Hoecker, ATLAS Spokesperson. “Since its beginning, ATLAS has strived to make its results fully accessible and reusable through open access archives such as [arXiv](https://arxiv.org/search/?query=ATLAS&searchtype=author&source=header) and [HepData](https://www.hepdata.net/search/?q=&collaboration=ATLAS). ATLAS has routinely released open data for educational purposes. Now, we’re taking it one step further — inviting everyone to explore the data that led to our discoveries.”
+
+Released under the [Creative Commons CC0 waiver](https://www.google.com/url?q=https://creativecommons.org/publicdomain/zero/1.0/legalcode.en&sa=D&source=docs&ust=1718970196670398&usg=AOvVaw1PABiSus03t0AxC3TYH5RL), ATLAS has made public all the data collected by the experiment during the 2015 and 2016 proton–proton operation of the LHC. This is approximately 65 TB of data, representing over 7 billion LHC collision events. In addition, ATLAS has released 2 billion events of simulated “Monte Carlo” data, which are essential for carrying out a physics analysis.
+
+External researchers, in particular, are encouraged to explore the ATLAS open data. “Along with the data, we have provided comprehensive documentation on several of our analyses, guiding users through our process step-by-step,” says Zach Marshall, ATLAS Computing Coordinator. “These guides provide first-hand experience of working on a real ATLAS result, allowing anyone to test our tools, and evaluate the systematic uncertainties associated with the result for themselves.”
+
+ATLAS traditionally collaborates with non-ATLAS scientists through [short-term associations](https://atlas.cern/Discover/Collaboration/External-Collaboration), granting them full access to ATLAS data, internal tools, and information. Through the open data, ATLAS researchers hope to further nurture this dialogue and collaboration. “In particular,” adds Zach, “we’d like to encourage phenomenologists and also computer scientists to explore our datasets, instead of relying on mock-ups.”
+
+Today’s release builds upon previous open data releases for educational use (in 2016 and 2020). “All of our open data releases are now available through the [ATLAS open data website](https://opendata.atlas.cern/),” says Dilia Portillo, ATLAS Outreach and Education Coordinator. “The website includes multi-level documentation, video tutorials and online tools aimed at the full-spectrum of users, from high school students to senior particle physics researchers. In addition, the software used to create the education-use open data has been released. This provides a seamless transition from the research open data to all the tutorials for outreach and education, including newly updated Higgs-boson discovery documentation. With a bit of time and dedication, you can go from being a relative novice to carrying out your own analysis.”
```

Comment:
One super minor request: to change "ATLAS Outreach and Education Coordinator" to ""ATLAS Outreach and Education co-Coordinator" here, and "ATLAS Computing Coordinator" above to "ATLAS Computing co-Coordinator".

Reply:
Done, coming in #3658